lib/generators.nix

```diff
@@ -307,4 +307,30 @@ rec {
 ${expr "" v}
 </plist>'';
 
+  /* Translate a simple Nix expression to Dhall notation.
+   * Note that integers are translated to Integer and never
+   * the Natural type.
+  */
+  toDhall = x:
```

Comment:
Ah, no, please add an option attrset here for the value and call it `v` instead of `x` to use the same scheme as the rest of this module.

Reply:
The attrset can be empty, but I want all generators to have the same interface  `toX { options } value`

Reply:
This also makes it a backwards-compatible change to introduce options later.

Reply:
That a good idea, we may want an option to convert integers to `Natural`.